docs/user/binning.rst

```diff
@@ -0,0 +1,146 @@
+.. _binning:
+
+Binning a Reduced Workspace in QuickNXS
+=======================================
+
+Overview
+--------
+
+Binning is an essential process in reflectivity reduction within QuickNXS, organizing raw data into manageable intervals for accurate analysis.
```

Comment:
Several binning options are available at various steps of the data reduction process.
An initial time-of-flight binning options is available to process the raw data, which is later converted into Q bins. A final Q rebinning is optionally available for user who want their reflectivity output in specific Q binning.